src/backend/TrafficCourts/Citizen.Service/Services/Impl/CitizenDocumentService.cs

```diff
@@ -138,11 +136,23 @@ public async Task<Guid> SaveFileAsync(IFormFile file, Dictionary<string, string>
             _logger.LogDebug("ticket-number value from metadata is empty");
         }
 
-        // Save file upload event to file history
-        SaveFileHistoryRecord fileHistoryRecord = new();
-        fileHistoryRecord.TicketNumber = ticketNumber;
-        fileHistoryRecord.Description = $"File: {file.FileName} was uploaded by the Disputant.";
-        await _bus.PublishWithLog(_logger, fileHistoryRecord, cancellationToken);
+        metadata.TryGetValue("notice-of-dispute-id", out string? noticeOfDisputeId);
+        if (string.IsNullOrEmpty(noticeOfDisputeId))
+        {
+            _logger.LogDebug("notice-of-dispute-id value from metadata is empty. Could not save document upload event to File History");
+        }
+
+        if (!string.IsNullOrEmpty(noticeOfDisputeId))
+        {
+            // Save file upload event to file history
+            SaveFileHistoryRecord fileHistoryRecord = new();
+            fileHistoryRecord.NoticeOfDisputeId = noticeOfDisputeId;
+            // TODO: This entry type is currently set to: "Document uploaded by Staff (VTC & Court)"
+            // since the original description: "File was uploaded by Disputant." is missing from the database.
+            // When the description is added to the databse change this
+            fileHistoryRecord.AuditLogEntryType = FileHistoryAuditLogEntryType.SUPL;
```

Comment:
Some audit log descriptions like this do not exist in the Oracle database currently so set some of those log messages to somehow related ones for now. Once those descriptions are added to the database with proper codes, we can update `FileHistoryAuditLogEntryType` enum for those file history descriptions.